component/src/test/java/org/wso2/extension/siddhi/io/mqtt/source/MqttSourceBrokerTest.java

```diff
@@ -85,8 +89,8 @@ public void receive(Event[] events) {
         fooStream.send(new Object[]{"WSO2", 55.6f, 100L});
         fooStream.send(new Object[]{"IBM", 75.6f, 100L});
         fooStream.send(new Object[]{"WSO2", 57.6f, 100L});
-        Thread.sleep(10000);
-        AssertJUnit.assertEquals(0, count);
+        SiddhiTestHelper.waitForEvents(waitTime, 3, count, timeout);
+        AssertJUnit.assertEquals(0, count.get());
```

Comment:
U r expecting 0 events?

Reply:
Since the broker was not available,no events were received. 

Anyway, realized that this test case is wrong,since the connection unavailable exception is never thrown,therefore i removed this test case in this commit  e8cbc53